config/settings.yml

```diff
@@ -1,3 +1,30 @@
+# the first endpoint_type listed must be for an local "online"
+# endpoint, and the default values listed for it will be used
+# in combination with the defined storage_roots to seed the db
+# with endpoints.
+endpoint_types:
+  local_online_nfs:
+    endpoint_class: online
+    endpoint_defaults:
+      endpoint_node: 'localhost'
+      recovery_cost: 1
+
 moab:
-  storage_roots: 'spec/fixtures'
+  # this is a list of named storage roots.  the names are used to seed the endpoints table.
+  storage_roots:
+    # name: '/location/on/disk'  # e.g.
+    fixtures: 'spec/fixtures'
   storage_trunk: 'moab_storage_root'
```

Comment:
it might be nice to add a comment about storage_trunk, such as "storage_trunk" is the parent directory of all the storage roots".

I would also like to see the `path_method` explicitly set to `:druid_tree` here (see https://github.com/sul-dlss/moab-versioning/blob/master/spec/unit_tests/stanford/storage_repository_spec.rb and https://github.com/sul-dlss/moab-versioning/blob/master/spec/spec_config.rb and